### PR TITLE
[IMP] sales: subscriptions follow up alerts add iap

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task:

https://www.odoo.com/web?debug=1#id=2057350&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@Felicious 

Can you review this small change on lines: 54-58?

Thanks, Tim 
:+1:

File: content/applications/sales/subscriptions/follow_up/automatic_alerts.rst [hunk 1]
```diff
@@ -51,6 +51,12 @@ customers, by email, after one month of subscription. To do so, go to :menuselec
       *Trigger date* and *Delay after trigger* need to be specified. And because the *Send an email
       to the customer* action was adopted, an *Email template* can be chosen.
 
+   .. note::
+      Sending a SMS test message in Odoo requires In-App Purchase (IAP) credit or tokens. For more
+      information on :abbr:`IAP (In-App Purchase)` visit
+      :doc:`../../../essentials/in_app_purchase`. For more information on sending SMS messages visit
```
Comment (Felicious) [inline content/applications/sales/subscriptions/follow_up/automatic_alerts.rst:57]:
```suggestion
      :doc:`../../../essentials/in_app_purchase`. For more information on sending SMS messages, visit
```

File: content/applications/sales/subscriptions/follow_up/automatic_alerts.rst [hunk 2]
```diff
@@ -51,6 +51,12 @@ customers, by email, after one month of subscription. To do so, go to :menuselec
       *Trigger date* and *Delay after trigger* need to be specified. And because the *Send an email
       to the customer* action was adopted, an *Email template* can be chosen.
 
+   .. note::
+      Sending a SMS test message in Odoo requires In-App Purchase (IAP) credit or tokens. For more
+      information on :abbr:`IAP (In-App Purchase)` visit
```
Comment (Felicious) [inline content/applications/sales/subscriptions/follow_up/automatic_alerts.rst:56]:
```suggestion
      information on :abbr:`IAP (In-App Purchase)`, visit
```

**Review by Felicious — APPROVED** [review]
Useful note block you added here, @tiku-odoo! :blush:

File: content/applications/sales/subscriptions/follow_up/automatic_alerts.rst [hunk 3]
```diff
@@ -51,6 +51,12 @@ customers, by email, after one month of subscription. To do so, go to :menuselec
       *Trigger date* and *Delay after trigger* need to be specified. And because the *Send an email
       to the customer* action was adopted, an *Email template* can be chosen.
 
+   .. note::
+      Sending a SMS test message in Odoo requires In-App Purchase (IAP) credit or tokens. For more
```
Comment (ksc-odoo) [inline content/applications/sales/subscriptions/follow_up/automatic_alerts.rst:55]:
```suggestion
      Sending a SMS text message in Odoo requires In-App Purchase (IAP) credit or tokens. For more
```

File: content/applications/sales/subscriptions/follow_up/automatic_alerts.rst [hunk 4]
```diff
@@ -51,6 +51,12 @@ customers, by email, after one month of subscription. To do so, go to :menuselec
...
+      information on :abbr:`IAP (In-App Purchase)`, visit
+      :doc:`../../../essentials/in_app_purchase`. For more information on sending SMS messages,
+      visit :doc:`../../../marketing/sms_marketing/essentials/sms_essentials`.
+
 As a result, this alert will send a rating survey after one month, to the customers who have
```
Comment (ksc-odoo) [inline content/applications/sales/subscriptions/follow_up/automatic_alerts.rst:60]:
```suggestion
As a result, this alert sends a rating survey after one month, to the customers who have
```

File: content/applications/sales/subscriptions/follow_up/automatic_alerts.rst [hunk 5]
```diff
@@ -51,6 +51,12 @@ customers, by email, after one month of subscription. To do so, go to :menuselec
...
 purchased that specific product. The survey will appear in the chatter of your respective
```
Comment (ksc-odoo) [inline content/applications/sales/subscriptions/follow_up/automatic_alerts.rst:61]:
```suggestion
purchased that specific product. The survey appears in the chatter of your respective
```

**Review by ksc-odoo — APPROVED** [review]
@tiku-odoo -- just finished my Final Review. Great job on this addition. Just a few minor things that need your attention, but approving now, because they are super quick fixes. Once it's ready, you can tag this for Tech Review!

**Comment by tiku-odoo** [comment]
@ksc-odoo 

This doc is ready for your review when you have a moment.

Thanks, 
Tim :+1:

**Comment by tiku-odoo** [comment]
@samueljlieber 

This document is ready for your review when you have a moment. 

Changes to lines: 54-58

Thanks,
Tim

**Review by samueljlieber — APPROVED** [review]
Hi @tiku-odoo, nice job this seems good to me! 
..
@robodoo r+

**Pull request closed without merging**